mwim.el

```diff
@@ -124,38 +124,63 @@ to the end of line."
           (goto-char eoc)))))
   (skip-chars-backward " \t"))
 
+(defun mwim-forward-line (arg)
+  "Move ARG - 1 lines forward (backward if ARG is negative).
+If ARG is nil or 1 do nothing."
+  (interactive "^p")
+  (setq arg (or arg 1))
+  (when (/= arg 1)
+    (let ((line-move-visual nil))
```

Comment:
IIUC, setting `line-move-visual` doesn't affect `forward-line`.  It
changes only the behaviour of `line-move`, so there is no need to set it
here.

BTW, this can simply be: ``(let (line-move-visual) ...)``